tests/applications/test_nexus_helpers.py

```diff
@@ -31,6 +41,26 @@ def ev44_generator() -> RandomEV44Generator:
     )
 
 
+def test_ymir_template_checksum() -> None:
+    """Test that the ymir template is updated.
+
+    ``ymir.json`` is modified to include detector data.
```

Comment:
I suggest renaming the modified version, so we can (if we ever want to) also use the original.

Reply:
@SimonHeybrock  Done...!